Engine/CMakeLists.txt

```diff
@@ -1,9 +1,8 @@
-# Please change to: 
-# ADD_LIBRARY(Engine
-#     <sources>
-# )
-# when engine is no longer header-only.
-# When you add a new file to the engine, please add it to the list below.
+# When you add a new file to the engine, please add it to the lists below.
+ ADD_LIBRARY(Engine
+     src/Window.cpp
```

Comment:
All source / header files in the custom target should be added here, and custom target can be removed.